### PR TITLE
client: Style change for upload file error

### LINKED ISSUE #8832: 'Upload File Error' modal should be updated with the new styling.
## Current Behavior
image

### PR DESCRIPTION
**Pull request by cansirin**
## Description

Currently "Update File Error" style was false according to new styling. This PR changes this to new Content Modal.
## Motivation and Context

Fixes #8832 
## How Has This Been Tested?

Manually
## Screenshots (if appropriate):

image
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: client/app/lib/commonviews/dnduploader.coffee [hunk 1]
```diff
@@ -59,14 +59,15 @@ module.exports = class DNDUploader extends KDView
...
       width        : 500
-      overlay      : yes
-      cssClass     : 'new-kdmodal'
+      cssClass     : 'content-modal'
       content      : "<div class='modalformline'>#{message}</div>"
+      overlay      : yes
       buttons      :
         Ok         :
+          title    : 'OK'
           style    : 'solid green medium'
```
Comment (hakankaradis) [inline client/app/lib/commonviews/dnduploader.coffee]: we dont have solid green class anymore
`style: 'GenericButton'`

**Comment by hakankaradis** [comment]
https://github.com/cansirin/koding/blob/b6ef260f261dd7893d238769de95606bdd0e8790/client/app/lib/commonviews/dnduploader.coffee#L82
can you please return after this line otherwise it will continue to upload files?

**Comment by cansirin** [comment]
updated /cc @hakankaradis

**Comment by hakankaradis** [comment]
👍

**Pull request merged**